app/frontend/src/components/designer/FormsTable.vue

```diff
@@ -78,7 +80,7 @@
           class="description-icon ml-2 mr-4"
           color="primary"
         >
-          {{ $t('trans.formsTable.description') }}
+          description
```

Comment:
I'm not sure what this is used for - screen readers? If so, should it be translated?